specification/suppressions.yaml

```diff
@@ -1643,13 +1643,6 @@
     - options.@azure-tools/typespec-ts.*
     - parameters.service-dir.default
   paths: [containerstorage/ContainerStorage.Management/tspconfig.yaml]
-- tool: TypeSpecValidation
-  reason: Remove specs contain incorrect or missing emitter options or parameters
-  rules: [SdkTspConfigValidation]
-  sub-rules:
-    - options.@azure-tools/typespec-go.flavor
-    - options.@azure-tools/typespec-java.namespace
```

Comment:
Why was this suppressed, if the code had it specified?  Did we have a bug in a tool we used to auto-add these suppressions?

Typically, we ignore unused suppressions, but maybe for SdkTspConfigValidation, you should fail on them, since we have so many now?